### DISCUSSION TITLE
Single Sign On for ByteStash Accounts

### DISCUSSION THREAD

**Discussion by jordan-dalby** [jordan-dalby/ByteStash#83] [Q&A] [answered]
Hi
Accounts are coming to ByteStash soon!

So... I have a question for everyone, what are you using for Single Sign On? I'm trying to determine which providers to implement initially.

**Comment by Massa-sapien** [comment]
pls, add docker secrets support, ( _FILE doesn't work) 
and scrolling preview in the main window of these few lines
really great work, great app for a programmer, thanks for your work!!

**Comment by philskents** [comment, marked as answer]
> Hi Accounts are coming to ByteStash soon!
> 
> So... I have a question for everyone, what are you using for Single Sign On? I'm trying to determine which providers to implement initially.

I use OIDC via Authentik for most of my services

**Comment by hamlm0** [comment]
I use EntraID, will it have the ability to assign groups against different code stash's?

**Reply by jordan-dalby** [reply]
Currently, no. OIDC is only used for accounts.

**Comment by dlew56** [comment]
Is there OIDC or SAML support yet?

**Reply by jordan-dalby** [reply]
Yes! OIDC is available.

**Reply by dlew56** [reply]
Thanks, I found the configuration here: https://github.com/jordan-dalby/ByteStash/wiki/Single-Sign%E2%80%90on-Setup
